lib/node_modules/@stdlib/math/base/special/cfloorn/docs/types/index.d.ts

```diff
@@ -20,43 +20,30 @@
 
 /// <reference types="@stdlib/types"/>
 
-import { ArrayLike } from '@stdlib/types/array';
+import { Complex128 } from '@stdlib/types/object';
```

Comment:
```suggestion
import { Complex128 } from '@stdlib/types/complex';
```

This type defn has been relocated.